elasticsearch/Dockerfile.in

```diff
@@ -0,0 +1,114 @@
+## EXCLUDE BEGIN ##
+#@follow_tag(openshift-ose-base:ubi8)
+FROM openshift-ose-base:v4.0-202104100031.20055 AS builder
+
+ARG upstream_code
+ENV upstream_code=upstream_code    
+ARG upstream_tarball=${upstream_code}.tar.gz    
+    
+COPY ${CI_ARCHIVE_ORIGIN_AGGREGATED_LOGGING} ${upstream_tarball}    
+RUN mkdir -p ${upstream_code} \    
+    && tar -xzf ${upstream_tarball} \    
+        --strip-components 1 \    
+        -C ${upstream_code}  
+## EXCLUDE END ##
+
+#@follow_tag(openshift-ose-base:ubi8)
+FROM openshift-ose-base:v4.0-202104100031.20055
+
+ENV BUILD_VERSION=${CI_CONTAINER_VERSION:-}
+ENV OS_GIT_MAJOR=${CI_X_VERSION:-}
+ENV OS_GIT_MINOR=${CI_Y_VERSION:-}
+ENV SOURCE_GIT_COMMIT=${CI_ORIGIN_AGGREGATED_LOGGING_UPSTREAM_COMMIT:-}
+ENV SOURCE_GIT_URL=${CI_ORIGIN_AGGREGATED_LOGGING_UPSTREAM_URL:-}
+
+LABEL \
+        License="GPLv2+" \
+        io.k8s.description="Elasticsearch container for EFK aggregated logging storage" \
+        io.k8s.display-name="Elasticsearch 6" \
+        io.openshift.tags="logging,elk,elasticsearch" \
+        vendor="Red Hat" \
+        name="openshift-logging/elasticsearch6-rhel8" \
+        com.redhat.component="logging-elasticsearch6-container" \
+        io.openshift.maintainer.product="OpenShift Container Platform" \
+        io.openshift.build.commit.id=${CI_ORIGIN_AGGREGATED_LOGGING_UPSTREAM_COMMIT} \
+        io.openshift.build.source-location=${CI_ORIGIN_AGGREGATED_LOGGING_UPSTREAM_URL} \
+        io.openshift.build.commit.url=${CI_ORIGIN_AGGREGATED_LOGGING_UPSTREAM_URL}/commit/${CI_ORIGIN_AGGREGATED_LOGGING_UPSTREAM_COMMIT} \
+        version=${CI_CONTAINER_VERSION}
+
+MAINTAINER OpenShift Development <dev@lists.openshift.redhat.com>
+
+EXPOSE 9200
+EXPOSE 9300
+USER 0
+
+ARG ES_ARCHIVE_URL
+ARG PROMETHEUS_EXPORTER_URL
+ARG OPENDISTRO_URL
+ARG OPENSHIFT_CI
+
+
+ENV ES_PATH_CONF=/etc/elasticsearch/ \
+    ES_HOME=/usr/share/elasticsearch \
+    ES_VER=6.8.1.redhat-00006 \
+    HOME=/opt/app-root/src \
+    INSTANCE_RAM=512G \
+    JAVA_VER=11 \
+    JAVA_HOME=/usr/lib/jvm/jre \
+    NODE_QUORUM=1 \
+    PROMETHEUS_EXPORTER_VER=6.8.1.0 \
+    OPENDISTRO_VER=0.10.0.4-redhat-00001 \
```

Comment:
need to make these same as
https://github.com/openshift/origin-aggregated-logging/pull/2109/files#diff-2644932b7f90cab52ba6d14e5dc4f970572bd8d64b2004fa6811fe10ecedbf53L33-L34